content/en/metrics/_index.md

```diff
@@ -111,6 +128,34 @@ Space aggregation splits a single metric into multiple timeseries by tags such a
 
 There are four aggregators that can be applied when using space aggregation: _sum_, _min_, _max_, and _avg_. Using the above example, say that your hosts are spread across four regions: us-east-1, us-east-2, us-west-1, and us-west-2. The hosts in each region need to be combined using an aggregator function. Using the _max_ aggregator would result in the maximum latency experienced across hosts in each region, while the _avg_ aggregator would yield the average latency per region.
 
+### Combining time series
+
+Time series are often combined together to produce a single representative time series. For example, you might want to see the average data received by the web servers in your infrastructure.
+
+Take two hosts submitting the same metric to Datadog:
+
+{{< img src="metrics/introduction/adding-by-host.png" alt="Two hosts send metrics to Datadog"  style="width:35%;">}}
+
+When you look at the data separated by host, `net.bytes_rcvd` is submitted at slightly different times:
+
+{{< img src="metrics/introduction/mismatched-time-series.png" alt="Mismatched Time Series" >}}
+
+### Breaking down the metric query
+
+In Datadog, the metric query looks like this:
+
+{{< img src="metrics/introduction/ui-query.png" alt="UI Query"  style="width:70%;">}}
+
+Looking at the JSON, the query can be broken out by space aggregation, metric name, scope, and grouping:
+
+{{< img src="metrics/introduction/color-query.png" alt="Query explained"  style="width:70%;">}}
+
+* **Scope** is the set of tags used to choose time series for the query.
+* **Grouping** is the set of tags over which to apply space aggregation.
+* **Time aggregation** is done implicitly, but can be set manually with the rollup function:
+
+{{< img src="metrics/introduction/color-query2.png" alt="Query explained"  style="width:70%;">}}
+
```

Comment:
This duplicates what we've already explained in detail starting at line 88, in "Anatomy of a Metric query" and uses outdated images of queries.